### PR TITLE
[stable/airflow] Add readinessProbe

### PR DESCRIPTION
**Pull request by khwj**
Signed-off-by: Khwunchai Jaengsawang <khwunchai.j@ku.th>

#### What this PR does / why we need it:
This PR adds `readinessProb` to Airflow web Deployment, this readiness prob is needed for some Ingress engine for health checking (e.g. with GCE ingress will not consider a backend is healthy since it’s keep waiting for `200` status code by querying `/` rather than `/health `).

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by gsemet — APPROVED** [review]
Oh great, I didn’t noticed they added the /health endpoint

/lgtm

**Comment by TomNeyland** [comment]
@gsemet @khwj
Looks like this just needs a trusted user to kick off the retest?

File: stable/airflow/requirements.lock [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
```
Comment (gsemet) [inline stable/airflow/requirements.lock:1]: I don’t think you should add this file

**Comment by khwj** [comment]
> @gsemet @khwj
> Looks like this just needs a trusted user to kick off the retest?

Think so, unfortunately I’m not a trusted user.

File: stable/airflow/requirements.lock [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- name: postgresql
+  repository: https://kubernetes-charts.storage.googleapis.com/
```
Comment (gsemet) [inline stable/airflow/requirements.lock:3]: Please remove this file

**Review by gsemet — CHANGES_REQUESTED** [review]
/approve cancel

**Comment by gsemet** [comment]
Damned, I am pretty sure this should not have been merged

**Pull request merged**